src/NonceManager.sol

```diff
@@ -40,5 +40,6 @@ abstract contract NonceManager is INonceManager, BaseAuthorization {
         if (!(nonceSequenceNumber[key]++ == seq)) {
             revert InvalidNonce();
         }
+        emit NonceSequenceNumberUpdated(key, seq++);
```

Comment:
Oh I thought we meant that we can just remove this event entirely? but only keep the invalidation event